lib/mdl/rules.rb

```diff
@@ -119,7 +119,7 @@
 rule 'MD007', 'Unordered list indentation' do
   tags :bullet, :ul, :indentation
   aliases 'ul-indent'
-  params :indent => 2
+  params :indent => 3
```

Comment:
This is changing a default setting, want to update or add a changelog entry to make sure this is communicated?

Since Kramdown will not recognize ordered list indentations of < 3 spaces, should this be configurable to a value < 3 spaces? If nothing else, please add a comment about why this is 3

Reply:
Yeah there's tests at a bunch of different spaces, but you're 100% correct that this needs a huge comment.

As to changelog, I usually do them all at once a release time, but I can add an entry here.

Reply:
And the rules doc should be updated

Reply:
2 will still work, as it did before. But then people will have to disable MD005 as mentioned above.

ULs have always worked at 2. But OLs only work at 3. I'm moving ULs to 3 to match. But people can still move it back to 2 if they want. It'll still work, just won't be consistent with OLs. And MD005 requires them to be the same